docs/components/optimize/userguide/additional-features/variable-labeling.md

```diff
@@ -0,0 +1,22 @@
+---
+id: variable-labeling
+title: Variable labeling
+description: Add a label for a variable in a process definition.
+---
+
+The variable labeling functionality allows users to add, update, and delete batches of variable labels so your data is more understandable by business users. This allows Optimize to display a variable's label instead of its original name anywhere the given process definition is being used. Some examples of that would be
+in reports, configuring filters, report grouping, dashboard filters, and event-based processes.
```

Comment:
Recommend: "when viewing and configuring reports, dashboards, or event-based processes"

Reply:
updated !